exodus/exodus/templates/home.html

```diff
@@ -45,7 +45,7 @@ <h4 style="display: none" id="results-title">{% trans "Results" %}</h4>
   <div class="col-lg-8 col-sm-12">
     <p class=mb-4>{% trans "εxodus analyzes Android applications in order to <b>list the embedded trackers.</b>" %}</p>
     <p class="mb-4">{% trans "A tracker is a piece of software <b>meant to collect data about you or your usages.</b> So, εxodus reports tell you what are the ingredients of the cake." %}</p>
-    <p>{% trans "εxodus does not decompile applications, its analysis technic is legal." %}</p>
+    <p>{% trans "εxodus does not decompile applications, its analysis method is legal." %}</p>
```

Comment:
I think we shouldn't change this unless we change the translation in the same time, otherwise we will break the current page.
So I would either add the translation now or remove this change and do it again once the translations are done.